.env.example

```diff
@@ -0,0 +1,13 @@
+AM_URL=$AM_URL
+APP_URL=$APP_URL
+API_URL=$API_URL
+DEBUGGER_OFF=$DEBUGGER_OFF
+DEVELOPMENT=$DEVELOPMENT
+REALM_PATH=$REALM_PATH
+JOURNEY_LOGIN=$JOURNEY_LOGIN
+JOURNEY_REGISTER=$JOURNEY_REGISTER
+DEVELOPMENT=$DEVELOPMENT
+PORT=$PORT
```

Comment:
Since we are further codifying this into our repo, we might want to distinguish what port this refers to. I think it's the API port, so we may want to label it as `API_PORT` to align with the other variables.